.github/workflows/mac_release.yml

```diff
@@ -20,18 +20,17 @@ jobs:
       # fail-fast: Default is true, switch to false to allow one platform to fail and still run others
       fail-fast: false
       matrix:
-        # TODO: macos-10.15 will be EOL on 8/30/22 and removed on 12/1/2022
-        os: [macos-10.15, macos-11, macos-12]
+        macos_dev_target: [10.15, 11.6, 12.1]
         include:
-        - os: macos-10.15
-          allow_failure: true
-          macos_dev_target: 10.15
-        - os: macos-11
+        - macos_dev_target: 10.15
+          os: macos-11
           allow_failure: false
-          macos_dev_target: 11.6
-        - os: macos-12
+        - macos_dev_target: 11.6
+          os: macos-11
+          allow_failure: false
+        - macos_dev_target: 12.1
+          os: macos-12
```

Comment:
run the MACOSX_DEPLOYMENT_TARGET=10.15 onto a macos-11 runner.

Reply:
Do we have to define the `macos_dev_target` variable first and then also put it inside the `matrix::include` scope too?

Reply:
Not sure I follow the question. The `macos_dev_target` **is** the matrix, and the include bit just adds other info for each.

This is functionally equivalent:

```
matrix:
  name: [macOS-10.15, macOS-11]
  include:
     - name: macOS-10.15
       os: macos-11
       macos_dev_tatget: 10.15
     - name: macOS-11
       os: macos-11
       macos_dev_tatget: 11.6
```

Reply:
I'm just saying I don't think you need to define the names first and then repeat them.  You can just list them out directly in the include block.  

```
matrix: 
  include: 
    - name: macOS-10.15 
      os: macos-11
      macos_dev_tatget: 10.15
    - name: macOS-11 
      os: macos-11 
      macos_dev_tatget: 11.6
```

Reply:
I didn't know that, thanks. I kinda like having the list(s) up top, makes it easier to see what's going on / how many runs this ends up producing.